templates/patterns/list_item/list_item.ui_patterns.yml

```diff
@@ -59,6 +59,14 @@ list_item:
       label: "Primary image"
       description: "Attributes of the image to render"
       preview:
-        src: "http://via.placeholder.com/400x400"
+        src: "https://placeimg.com/1200/900/arch"
         alt: "Alternative text for image"
-
+    infos:
+      type: "array"
+      label: "Infos"
+      description: "A list of infos of the page."
+      preview:
+        - text: "Monday 8 February"
```

Comment:
We already have a date field in the pattern. I'm wondering if we should try and add the value on the date field to the infos whenever infos is used. 
Also having both meta and infos is a bit confusing. I know it comes from ECL but as it stands there is no way to know what this fields mean without looking at the code.

Reply:
I understand I will change the preview to be consistent with ECL2.
https://ec.europa.eu/component-library/ec/templates/compositions/content-item/code/